### PR TITLE
Zultron/2020 05 12 lcnc 709 single step

### PR DESCRIPTION
**Pull request by zultron**
This is a test case and fix for #709.  The commit log explains the problem and solution.

Thanks, @andypugh, for not only figuring out how to reproduce this, but also locating where the problem was in the code.  That made it a whole lot easier to fix!

### CODE REVIEW

File: tests/single-step/test-ui.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```
Comment (dwrobel) [inline tests/single-step/test-ui.py:1]: Please change `python` -> `linuxcnc-python`.

Comment (zultron) [reply tests/single-step/test-ui.py]: Done.

**Comment by zultron** [comment]
Checks have passed.

**Comment by andypugh** [comment]
Thanks

**Pull request merged**